CharlieBackend.Panel/Utils/HttpUtil.cs

```diff
@@ -76,16 +76,22 @@ public async Task<HttpResponseMessage> DeleteAsync(string url)
             return responseMessage;
         }
 
-        public async Task EnsureSuccessStatusCode(HttpResponseMessage httpResponse)
+        public async Task<string> EnsureSuccessStatusCode(HttpResponseMessage httpResponse)
         {
             if (!httpResponse.IsSuccessStatusCode)
             {
                 string apiStringResponse = await httpResponse.Content.ReadAsStringAsync();
 
-                var apiResponseMessage = JsonConvert.DeserializeObject<ErrorDto>(apiStringResponse).Error.Message;
+                string apiResponseMessage;
+                if (apiStringResponse != "")
+                    apiResponseMessage = JsonConvert.DeserializeObject<ErrorDto>(apiStringResponse).Error.Message;
+                else
+                    apiResponseMessage = ErrorsConstants.EmptyApiStringResponse;
 
-                throw new HttpStatusException(httpResponse.StatusCode, apiResponseMessage);
+                return apiResponseMessage;
```

Comment:
curly brackets for "if else" body, 
if condition replace "" to string.Empty,
and it is possible to simplify it as inline return or assigned to the variable "apiResponseMessage" 
```
return apiStringResponse != string.Empty 
          ? JsonConvert.DeserializeObject<ErrorDto>(apiStringResponse).Error.Message
          : ErrorsConstants.EmptyApiStringResponse;
```